packages/medusa/src/services/__tests__/product-category.ts

```diff
@@ -0,0 +1,49 @@
+import { IdMap, MockRepository, MockManager } from "medusa-test-utils"
+import ProductCategoryService from "../product-category"
+
+describe("ProductCategoryService", () => {
+  describe("retrieve", () => {
+    const productCategoryRepository = MockRepository({
+      findOne: query => {
```

Comment:
I dont think you have linting on @riqwan, these parameters for arrow functions should be wrapped in parenthesis. We have an `eslintrc.js` in the repo that you should be able to use 😄

Reply:
I don't, thanks for the shout. I'll set it up! :D